theme/ckeditor5-ui/components/list/list.css

```diff
@@ -79,3 +84,81 @@
 	width: 100%;
 	background: var(--ck-color-base-border);
 }
+
+.ck .todo-list {
+	list-style: none;
+}
+
+.ck .todo-list > li {
+	margin-bottom: 5px;
+}
+
+.ck .todo-list > li > .todo-list {
+	margin-top: 5px;
+}
+
+/*
+ * Let's hide native checkbox and make a fancy one.
+ *
+ * Note: Checkbox element cannot be hidden using `display: none` or `visibility: hidden`.
+ * It has to be clickable to not steal the focus after clicking on it.
+ */
+.ck .todo-list__checkmark input[type='checkbox'] {
+	display: block;
+	width: 100%;
+	height: 100%;
+	opacity: 0;
+	margin: 0;
+}
+
+.ck .todo-list__checkmark {
+	cursor: pointer;
+	width: var(--ck-todo-list-checkmark-size);
+	height: var(--ck-todo-list-checkmark-size);
+	position: relative;
+	display: inline-block;
+	vertical-align: middle;
+	left: -23px;
```

Comment:
Can we use `calc()` and `--ck-todo-list-checkmark-size` here somehow? Because I assume these two have a lot in common.

Reply:
Nope, it's not the same thing, this offset is not related to the checkmark size, see left offset based on `--ck-todo-list-checkmark-size`:
<img width="490" alt="Screenshot 2019-09-18 at 09 26 41" src="https://user-images.githubusercontent.com/10219857/65125829-70d99280-d9f6-11e9-8b76-3ff4e99bee3e.png">

Anyway, I've found out the "tick" issue :/